CMakeLists.txt

```diff
@@ -210,39 +210,36 @@ set(ADMIN_DIR services/user/AdminSys/ui:AdminSys_js)
 set(COMMON_LIB_DIR services/user/CommonSys/common/packages/common-lib/:CommonSysCommonLib_js)
 set(COMMON_LOADER_DIR services/user/CommonSys/common/packages/wasm-loader/:CommonSysLoaderLib_js)
 
-function(add_js_target ENTRY OUTPUT_FILE)
-    string(REGEX REPLACE "^([^:]+):([^:]+)$" \\1 PATH ${ENTRY})
-    string(REGEX REPLACE "^([^:]+):([^:]+)$" \\2 DEP ${ENTRY})
+function(add_js_target ROW OUTPUT_FILE)
+    string(REGEX REPLACE "^([^:]+):([^:]+)$" \\1 PATH ${ROW})
+    string(REGEX REPLACE "^([^:]+):([^:]+)$" \\2 TargetName ${ROW})
```

Comment:
Why `TargetName` and not `TARGET_NAME` like other variable names? Is it just a hint that the values are in PascalCase? (`CommonSys_js`, `TokenSys_js`, etc.)?

Reply:
Yeah it's sort of a convention that target names are PascalCase. My thinking was that, even though it's a variable, it contains the name of a target and so I wanted to indicate that by giving the variable PascalCase.

But... upon reflection, I don't actually like this. It's a variable, should be caps.

Reply:
fixed